### PR TITLE
Release: Qubes OS 4.2.0-rc2

### PR DESCRIPTION
**Pull request by andrewdavidwong**

### CODE REVIEW

File: 2023-08-XX-qubes-os-4-2-0-rc2-available-for-testing.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+If you're willing to [test](/doc/testing/) this new release candidate, you can help us improve the eventual stable release by [reporting any bugs you encounter](/doc/issue-tracking/). We encourage experienced users to join the [testing team](https://forum.qubes-os.org/t/joining-the-testing-team/5190).
+
+A full list of issues affecting Qubes 4.2.0 is available [here](https://github.com/QubesOS/qubes-issues/issues?q=is%3Aissue+label%3Aaffects-4.2). We strongly recommend [updating Qubes OS](/doc/how-to-update/) immediately after installation in order to apply all available bug fixes.
+
+## Upgrading to Qubes 4.2.0-rc2
+
+In-place upgrades from Qubes 4.1 to Qubes 4.2 are now implemented and ready for testing! As always, we strongly recommend [making a full backup](/doc/how-to-back-up-restore-and-migrate/) beforehand.
+
+Current Qubes 4.2.0-rc1 systems should be [updated normally](/doc/how-to-update/).
```
Comment (marmarek) [inline 2023-08-XX-qubes-os-4-2-0-rc2-available-for-testing.md:47]: Maybe worth mentioning updated templates here, compared to rc1?

**Review by marmarek — CHANGES_REQUESTED** [review]

File: 2023-08-XX-qubes-os-4-2-0-rc2-available-for-testing.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Thank you to everyone who tested 4.2.0-rc1! Due to your efforts, this new release candidate includes fixes for several bugs that were present in the first release candidate.
+
+If you're willing to [test](/doc/testing/) this new release candidate, you can help us improve the eventual stable release by [reporting any bugs you encounter](/doc/issue-tracking/). We encourage experienced users to join the [testing team](https://forum.qubes-os.org/t/joining-the-testing-team/5190).
+
+A full list of issues affecting Qubes 4.2.0 is available [here](https://github.com/QubesOS/qubes-issues/issues?q=is%3Aissue+label%3Aaffects-4.2). We strongly recommend [updating Qubes OS](/doc/how-to-update/) immediately after installation in order to apply all available bug fixes.
+
+## Upgrading to Qubes 4.2.0-rc2
+
+In-place upgrades from Qubes 4.1 to Qubes 4.2 are now implemented and ready for testing! As always, we strongly recommend [making a full backup](/doc/how-to-back-up-restore-and-migrate/) beforehand.
```
Comment (marmarek) [inline 2023-08-XX-qubes-os-4-2-0-rc2-available-for-testing.md:45]: upgrade docs: https://github.com/QubesOS/qubes-doc/pull/1336
maybe link here too?

File: 2023-08-XX-qubes-os-4-2-0-rc2-available-for-testing.md (hunk 1 above)
Comment (andrewdavidwong) [reply 2023-08-XX-qubes-os-4-2-0-rc2-available-for-testing.md:47]: They're already listed [above](https://github.com/QubesOS/qubes-posts/pull/117/files#diff-09d897cff3fc9a9167181ea07660ceff91d054a93b63f5586d6b4daadb081cc7R14-R15) in the "what's new" section. I see a few options:

1. Make that section RC-specific (i.e., "What's new in Qubes 4.2.0-rc2?"). It currently just targets the stable release, even though the contents of the list do get updated with each RC, if appropriate.
2. Duplicate the content (i.e., list all changes up there and the subset that's different between RCs down here).
3. A third option that I forgot.

(2) seems messy and unnecessarily verbose to me, so I'm leaning toward (1).

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of 3af676b0c126deec8a01eab878994cc49917a21b
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmTtJwkACgkQ8yiUvpaE
k4qPmA/+P4WQ61yD1a7SgyspOoSfJeD8osccT3dV7bzrgiNuSuyCToEjfW35kiLZ
hDP9ssryOk+f5R0hQfF6qFTAVKJa4w95pJ5XL/hcrj/kxM2DNyLjnbsjL3XARTb+
ebuNut7wdayLCEIdoaZEprLeURdyB2ao4SLbnpAkUu0A/YR8mhdchgaBP6503tvT
JVgi8lymDf7iW3i+Cn1xNoktZnJwtlyAsFReCFjqYLuO3Ymw7St9qhiUXC3NU5N2
Ue+zYp9LXbwvrN6YDQP6qBk9zz8aGWPA6a2LmlX0RpCq8P6sU+v5pfOWNsK+ph+d
1oKOape82w8DTDWh8rda9nFPPi+ZNragA/5dcmPahfZ5OhTZhXtmAF7bVPROvdX2
IGKRHEugUAcQnXDof9PmzkHTqLZGFakj4vsyUY2UrWLcauVK9ffv3js1UuY47P4F
wfq0tsm2lBDe1DKf5W8oXDZa0jwILqsCMvW0xNtaeXVWrL3AlvgY1VGuUnRyEtAv
njW9lfAGY7htxba3yygclMsndcPXovnu4KZP5QHRoasLqolYe/nZE39JSHKPMzX/
2B6nsLW4LAW5CTd+1dt/RqIh3xBZcnesbjVKx53uX3Xb3zfMmDh+EL8Mm/wQm6Yu
Ksj9GV6ErRmglfenWwC2uqFV3v8ZN+aQqOXHZPnRIBfrAKLNVes=
=uHiK
-----END PGP SIGNATURE-----
```

**Comment by marmarek** [comment]
It's still Monday in UTC! ;)

**Pull request merged**